scss/_variables.scss

```diff
@@ -1183,8 +1183,12 @@ $form-range-thumb-border-radius:           50% !default;
 $form-range-thumb-box-shadow:              0 .1rem .25rem rgba($black, .1) !default;
 $form-range-thumb-focus-box-shadow:        null !default;
 $form-range-thumb-focus-box-shadow-width:  $input-focus-width !default; // For focus box shadow issue in Edge
-$form-range-thumb-active-bg:               null !default;
-$form-range-thumb-active-border:           $component-active-bg !default; // Boosted mod
+$form-range-thumb-hover-bg:                $black !default; // Boosted mod
```

Comment:
⚠️ It doesn't work on black background

![Screenshot 2022-12-26 at 10 07 39](https://user-images.githubusercontent.com/17381666/209529579-eea07aa8-4eef-4740-813c-b63a0a8b778b.png)

Reply:
Do you mean that a `.form-range-inverted` should be added to this PR ?

Reply:
If we absolutely need a `.form-range-inverted` no. But on our current main branch, the dark variant worked well thanks to the black border and white bg even when dragging the handle bar.

Reply:
`var(--#{$prefix}link-color)` or maybe just `currentColor` might do the trick here ?

Reply:
Done with last commit